script.module.tzdata/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tzdata" name="tzdata" version="2021.5+matrix.1" provider-name="pganssle">
```

Comment:
if this won't be available below matrix, no need for the local version identifier